docs/source/EccFramework.rst

```diff
@@ -0,0 +1,201 @@
+ECC Framework: Automatic Implementation and Evaluation of Error-Correcting Codes for Quantum Computing
+======================================================================================================
+
+The QECC library offers means for automatic implementation and evaluation of error-correcting codes for quantum computing. More precisely, the library allows to automatically apply different error correction schemes to quantum circuits provided as openqasm files. The "protected" quantum circuits can then be exported again in the form of openqasm files or can be directly used for noise-aware quantum circuit simulation. For the latter case, we also provide a wrapper script which makes use of the provided framework to apply error correction schemes to circuits and directly simulate those circuits using qiskit.
```

Comment:
Furthermore, I thought the code also accepts Qiskit QuantumCircuit objects as input?

Reply:
Indeed. I updated the documentation.